### PR TITLE
Linux ARM compatibility

### PR DESCRIPTION
**Pull request by tbutter**
A patch for Linux ARM compatibility (Rasperry PI).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -12,7 +12,7 @@ repositories {
 
 
 dependencies {
-	compile 'net.java.dev.jna:jna:3.5.2'
+	compile 'net.java.dev.jna:jna:4.1.0'
```
Comment (EtiennePerot) [inline build.gradle]: Please don't change this, it causes issues. https://github.com/EtiennePerot/fuse-jna/issues/28

**Comment by EtiennePerot** [comment]
Thanks! Nice work.
Have you thoroughly tested most filesystem operations, or just a cursory check?

**Comment by EtiennePerot** [comment]
Also, please update the README file about this (Compatibility section).

File: build.gradle (hunk 1 above)
Comment (tbutter) [inline build.gradle]: the issue only appears with 4.0.0 in my tests.

**Comment by tbutter** [comment]
I tested it with my filesystem (https://github.com/tbutter/gyingpan) using the most common FS operations on a Rasperry Pi for several hours copying files, stat, md5sum, etc. I did not test xattr, but I think it shouldn't be affected by that change.

File: build.gradle (hunk 1 above)
Comment (EtiennePerot) [inline build.gradle]: Please don't change it in this pull request anyway. This pull request is about adding Linux ARM compatibility, and this change has nothing to do with that. You can open another pull request to change it instead and I'll accept it after testing that it was a 4.0.0 issue (I never really investigated it...)

Comment (tbutter) [inline build.gradle]: Unfortunately a higher version IS needed. According to the changelog 3.5.2 should support ARM but the binaries on mavencentral do not contain com/sun/jna/linux-arm/libjnidispatch.so. It is in 4.1.0.

Comment (EtiennePerot) [inline build.gradle]: Wow, you're right. Alright, thanks!

**Comment by EtiennePerot** [comment]
Do you mind if I add your project to the list of projects using fuse-jna in the README file?

**Comment by tbutter** [comment]
No, please add it.

**Pull request merged**